src/applications/third-party-app-directory/actions/index.js

```diff
@@ -27,39 +33,55 @@ export const fetchResultsSuccess = response => ({
 });
 
 // ============
-// Redux Thunks
+// Fetch Scopes
 // ============
-export const fetchResultsThunk = options => async dispatch => {
-  const { hideFetchingState, page = 1, perPage = 10 } = options;
 
+export const fetchScopesAction = () => ({
+  type: FETCH_SCOPES,
+});
+
+export const fetchScopesFailure = error => ({
+  error,
+  type: FETCH_SCOPES_FAILURE,
+});
+
+export const fetchScopesSuccess = (response, scopeCategory) => ({
+  response,
+  scopeCategory,
+  type: FETCH_SCOPES_SUCCESS,
+});
+
+// ============
+// Redux Thunks
+// ============
+export const fetchResultsThunk = () => async dispatch => {
   // Change the `fetching` state in our store.
   dispatch(
     fetchResultsAction({
-      hideFetchingState,
-      page,
+      hideFetchingState: true,
     }),
   );
 
   try {
     // Attempt to make the API request to retreive results.
-    const response = await fetchResultsApi({
-      page,
-      perPage,
-      mockRequest: true,
-    });
-
-    // Track the API request.
-    // if (trackSearch) {
-    //   recordEvent({
-    //     event: 'third-party-apps-search',
-    //     'third-party-apps-category': category || undefined,
-    //     'third-party-apps-platform': platform || undefined,
-    //     'third-party-apps-number-of-search-results': response?.totalResults,
-    //   });
-    // }
+    const response = await fetchResults({});
 
     // If we are here, the API request succeeded.
     dispatch(fetchResultsSuccess(response));
+    const uniqueScopes = uniq(
```

Comment:
Fetching scopes here instead of on additional info toggle to prevent user from having to see the scopes render